samples/drumthumper/README.md

```diff
@@ -0,0 +1,37 @@
+**DrumThumper**
+==========
+Oboe playback sample app.
+
+## Abstract
+**DrumThumper** (apolgies to [Chumbawamba](https://www.youtube.com/watch?v=2H5uWRjFsGc)) is a "Drum Pad" app which demonstrates best-practices for low-latency audio playback using the Android **Oboe** API.
+**DrumThumper** consists of a set of trigger pad widgets and an optional UI for controlling the level and stereo placement of each of the virtual drums.
```

Comment:
The mixing and WAV reader are important.  Maybe add something about "Drum samples are read from WAV files into memory. The samples are mixed into a single audio stream."